action.yml

```diff
@@ -3,14 +3,29 @@ description: 'A GitHub Action to run a UI test based on user instructions'
 author: 'foreai-co'
 
 inputs:
+  website:
```

Comment:
how about calling this argument 'url' rather than 'website'? website sounds a bit limited - but either is fine.

Reply:
Done.